modules/serverless-logic-upgrade-1.36-finalizing-job-service.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.35-to-1.36
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-1.36-finalizing-job-service_{context}"]
+= Finalizing the Job Service upgrade
+
+After the {ServerlessOperatorName} is upgraded to version 1.36.0 you can optionally delete the old `ReplicaSet`.
+
+.Procedure
```

Comment:
@kaldesai 

For the jobs service we should apply the same pattern as for the Data Index
:point_down: 
![image](https://github.com/user-attachments/assets/6a5bdb69-0320-4b6b-ba3b-6993f0609ff8)